### ISSUE TITLE
Handle tagged AND untagged COSE objects

### ISSUE THREAD

**Issue by martin-lindstrom** [ehn-digital-green-development/ValidationCore#4]
I received some comments from developers that had been looking at your solution that you can't handle when the Cose_Sign1 isn't tagged. See my issue https://github.com/ehn-digital-green-development/hcert-spec/issues/69.

**Comment by asitdm** [comment]
Hi,

could your verify if the changes of [https://github.com/ehn-digital-green-development/ValidationCore/issues/3](https://github.com/ehn-digital-green-development/ValidationCore/issues/3) resolve your issue as well?
Support for other COSE objects like Cose_Sign is currently planned for future versions.

**Comment by DKvarn** [comment]
Yes it's now fixed with the latest fix in #3 , you can close this now

**Comment by asitdm** [comment]
Thank you for the pull request, already tested and merged it.
